packages/ember-intl/index.js

```diff
@@ -115,8 +117,8 @@ module.exports = {
       outputPath: options.outputPath ?? outputPath,
       requiresTranslation,
       stripEmptyTranslations,
-      verbose: !this.isSilent,
```

Comment:
this.isSilent points at nothing, so ended up being always true

Reply:
It looks like the variable `isSilent` is [a remnant from `v4.x` or earlier](https://github.com/ember-intl/ember-intl/blob/v4.0.1/index.js#L61). We can definitely remove it.